website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Edit Permissions | PUT
+        Rule([
```

Comment:
Why is the GET `/wiki/permissions/` and the PUT `/wiki/settings/`? It would probably make sense to put them both into `/wiki/settings/` and rename the view while you're there. That way, it'll be flexible for future changes to node wiki settings.